lib/tasks/flag_missed.rake

```diff
@@ -1,7 +1,7 @@
 desc 'Flag not checked out reservations starting yesterday as missed'
 task flag_missed: :environment do
   new_missed = Reservation.where(
-    'start_date <= ?', Time.zone.today - 1.day).reserved
```

Comment:
I tend to prefer referencing all dates by `Time.zone.today` (vs using `yesterday` or `tomorrow` just for clarity); was there a reason why you switched this?